.github/workflows/create-versioning-pr.yaml

```diff
@@ -1,20 +1,23 @@
-name: Version Kubernetes Agent chart
+name: Version Charts 
 
 on:
   push:
     branches:
     - main
     - release/kubernetes-agent/v*
+    - release/octopus-deploy/v*
+    - octopus-v1
     paths:
     - charts/kubernetes-agent/**
+    - charts/octopus-deploy/**
     - .github/workflows/create-versioning-pr.yaml
     
 jobs:
   publish:
     runs-on: ubuntu-latest
 
     # When a changesets generated Versioning PR is merged, we want to publish the version as a tag
-    if: ${{ startsWith(github.event.commits[0].message, 'Version Kubernetes Agent Chart') }}
+    if: ${{ startsWith(github.event.commits[0].message, 'Version Charts') }}
```

Comment:
The tagging here is only set up for the `kubernetes-agent/{version}`... I _assume_ you want to tag it `octopus-deploy/{version}`?

Reply:
Yeah - I'll add a step to do so...

Reply:
We moved this over to a new workflow and only trigger it when changes are made to the agent. This may fail on the next versioning run when this gets merged, as it'll run since we changed the workflow itself.